src/data/causeSpecificJSONData/cats.json

```diff
@@ -0,0 +1,55 @@
+{
+  "path": "cats",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "FF6A08",
+        "secondary": "29BEBA",
+        "secondaryShadow": "15AAA6",
+        "background": "FAEFE1"
+      }
+    },
+    "metadata": {
+      "title": "Tab for Cats - Home",
```

Comment:
@kmjennison am i missing anything in the metadata section?  This was based on what I saw in cats

Reply:
That's all I can think of for now. I know this is a draft, but just some feedback:
* Assuming `causeSpecificKeywords` is joined with a generic list of keywords, then don't need "charity"
* Probably don't need `titleTemplate`. Don't think we'll customize it per page?
* Typo: `cannonicalUrl` --> `canonicalUrl`. Though, I'd suggest we add a `pagePath` key that's used in Gatsby page generation and use it for canonical URL (don't want them to get out of sync).